packages/rn-tester/Podfile.lock

```diff
@@ -501,6 +501,33 @@ SPEC CHECKSUMS:
   Folly: b73c3869541e86821df3c387eb0af5f65addfab4
   glog: 40a13f7840415b9a77023fbcae0f1e6f43192af3
   OpenSSL-Universal: 8b48cc0d10c1b2923617dfe5c178aa9ed2689355
+<<<<<<< HEAD:packages/rn-tester/Podfile.lock
```

Comment:
Merge conflict still in this file, doesn't this make the lockfile invalid?

Reply:
Resolved them 😄